project.godot

```diff
@@ -215,7 +215,6 @@ translations=PoolStringArray( "res://locale/en.po", "res://locale/fr.po", "res:/
 [logging]
 
 file_logging/enable_file_logging=true
-file_logging/log_path="user://logs/log.txt"
```

Comment:
This needs to be reverted.

Reply:
that's just a bad merge cause when i just come back on the code I got this line replace : file_logging/max_log_files=8

Reply:
this change is still here

Reply:
I don't understand ? I am  supposed to replace 
file_logging/log_path="user://logs/log.txt" into 
file_logging/max_log_files=8
or delete both

Reply:
You need to copy the content that is on master branch so that this PR no longer has any changes shown to project.godot (https://github.com/Revolutionary-Games/Thrive/pull/1785/files)